client/layout/poll-invitation/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import snakeCase from 'lodash/snakeCase';
+import includes from 'lodash/includes';
+import memoize from 'lodash/memoize';
+let debug = require( 'debug' )( 'calypso:poll-invitation' );
```

Comment:
Two remarks:

- Generally we prefer to use `const` as much as possible over `let`, as the latter is a good indication that the value of the variable is reassigned at some point in time.
- The general approach we've taken with modules that export a function to be invoked is:

```js
/**
 * External dependencies
 */
import debugModule from 'debug';

/**
 * Internal dependencies
 */
import userModule from 'lib/user';

/**
 * Module variables
 */
const debug = debugModule( 'calypso:poll-invitation' );
const user = userModule();
```